src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesVsPatterns.cs

```diff
@@ -174,13 +174,13 @@ void Test(object? x)
 }
 ");
             c.VerifyDiagnostics(
-                // (7,18): error CS0150: A constant value is expected
+                // 0.cs(7,18): error CS0150: A constant value is expected
```

Comment:
> 0.cs [](http://example.com/codeflow?start=18&length=5)

It looks like there are no meaningful changes in this file. I recommend restoring it to its original state.